src/language/class-modifiers.md

```diff
@@ -194,15 +205,16 @@ class MockVehicle          // ERROR: Cannot be implemented
 
 ## `sealed`
 
-Use the `sealed` modifier to create a known, enumerable set of subtypes, so that
-a switch over those types can be statically ensured to be [_exhaustive_][exhaustive].
+To create a known, enumerable set of subtypes, so that a switch over those types
+can be statically ensured to be [_exhaustive_][exhaustive], use the `sealed` modifier.
 
 The `sealed` modifier prevents a class from being extended or implemented outside
-of the library where the class is defined. The compiler is aware of any possible
-direct subtypes because they can only exist in the same library.
+of the library where the class is defined. They also prevent construction,
+and are therefore implicitly [abstract](#abstract).
```

Comment:
**suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice). Not certain of my construction here, so you can ignore.
```suggestion
of the library where you define the class. You can't construct these classes,
so they have implicit [abstraction](#abstract).
```